tests/unity/example_test/src/foo/foo.h

```diff
@@ -10,44 +10,14 @@
 #include <toolchain/common.h>
 
```

Comment:
Now that you move several test functions out of this file, could you please add a description as to what the purpose of this file now is.
Similar to what you do here:
https://github.com/nrfconnect/sdk-nrf/blob/42e3f1a5f33bd088d8ac429ce49d60fd7fff8478/tests/unity/wrap_test/src/fns/fns.h#L10-L11

Reply:
I don't know the intention of it, but I can always make a guess 😉

Reply:
Found out it's mentioned in our [docs](https://developer.nordicsemi.com/nRF_Connect_SDK/doc/latest/nrf/ug_unity_testing.html), so will base it on that.

But wouldn't it be better to have a readme than a comment?

Reply:
not sure in this case.
The readme is something intended for end-users to read and to go into the docs.

This is more a comment so that those editing the files or fixing issues with the scripts knows which test files to update correctly.

Like you do in this PR :)

Reply:
I'm not sure I quite understand. Except for the `static inline` function (that was left by mistake), it is just a regular header file declaring the functions in `foo.c`. Does it need an explanation? Or were you thinking in case someone is looking for the wrapper snippets that was in `foo.h` before? To let them know it is in the `wrap_test` folder now.
I'm about to convert the `static inline` function to a regular function declaration and define it in `foo.c`. So there will be no signs after the wrapping tests left in the example_test. Does it really need an explanation? The `example_test` will be used as a template for unit testing, so it will look strange if we have those kind of comments in the template in my opinion.

Reply:
I took an extra look at `foo.h/c` and I see you removed everything that was actually testing something.

So `foo.h/c` is a pure example, so guess no special comment is needed after all.